src/components/AddNewCommentForm.jsx

```diff
@@ -127,8 +127,32 @@ class AddNewCommentForm extends React.Component {
     }
 
     render() {
+        const isMac = window.navigator.platform.includes('Mac');
+        const isMeta = (key) => (isMac && key.key === 'Meta') || (!isMac && key.key === 'Control');
+        const markMetaKeyUp = (locality) => (key) => {
+            if (isMeta(key)) {
+                locality.metaDown = false;
+            }
+        };
+        const performActionOnMetaEnter = (locality, action) => (key) => {
+            if (isMeta(key)) {
+                locality.metaDown = true;
+            } else if (locality.metaDown && key.key === 'Enter') {
+                action();
```

Comment:
Can we remove the `action` and `locality` parameters. Having these is great if we would be able to reuse these functions in other context. Assuming they are private for this component, it might be simpler to just call the real methods. Or even to move these as functions on the component rather than local members